tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -2079,8 +2079,6 @@ def test_errored_user_sessions_clamped_to_zero(self):
     def test_healthy_user_sessions(self):
```

Comment:
Code LGTM, it would be nice to add more tests around changes were made.

Reply:
i agree. @jjbayer I also realized, you need to update metrics extraction in sentry's testharness

Reply:
> you need to update metrics extraction in sentry's testharness
Yep, working on it